Makefile

```diff
@@ -24,10 +24,13 @@ bundle-docs-clean:
 
 bundle-docs: bundle-docs-clean
 	cd docs && \
-	make html && \
+	make html SPHINXOPTS="-W --keep-going" && \
 	doc2dash --name pyteal --index-page index.html --online-redirect-url https://pyteal.readthedocs.io/en/ _build/html && \
 	tar -czvf pyteal.docset.tar.gz pyteal.docset
 
+# The `SPHINXOPTS="-W -keep-going"` comes from https://www.sphinx-doc.org/en/master/man/sphinx-build.html#cmdoption-sphinx-build-W
+# also check link https://stackoverflow.com/questions/38048945/how-to-turn-warnings-into-errors-when-building-sphinx-documentation-with-setupto
+
```

Comment:
@ahangsu Optional - Feel welcomed to resolve as is:  Since we don't annotate other command invocations, I don't see a _need_ to do so here.  I think `SPHINXOPTS` provides a keyword for folks to google as needed.

I raise the comment because the `Makefile` is liable to grow quite large in size.  I do _not_ feel strongly and welcome you to resolve as is if you prefer.